src/hardware_trigger.hpp

```diff
@@ -36,6 +36,31 @@ class HardwareTrigger
 		N_DIGITAL_AND_ANALOG = 7,
 		N_DIGITAL_XOR_ANALOG = 8,
 	};
+	/*enum mode {
```

Comment:
Will `enum mode` still be needed in the future?

Reply:
fixed - enabled the newer enum